exporter/datadogexporter/metrics_exporter.go

```diff
@@ -187,18 +212,41 @@ func (exp *metricsExporter) PushMetricsData(ctx context.Context, md pmetric.Metr
 	if src.Kind == source.AWSECSFargateKind {
 		tags = append(tags, exp.cfg.HostMetadata.Tags...)
 	}
-	ms, sl, sp := consumer.All(exp.getPushTime(), exp.params.BuildInfo, tags)
-	ms = metrics.PrepareZorkianSystemMetrics(ms)
 
-	err = nil
-	if len(ms) > 0 {
-		exp.params.Logger.Debug("exporting payload", zap.Any("metric", ms))
-		err = multierr.Append(
-			err,
-			exp.retrier.DoWithRetries(ctx, func(context.Context) error {
-				return exp.client.PostMetrics(ms)
-			}),
-		)
+	var sl sketches.SketchSeriesList
+	var sp []pb.ClientStatsPayload
+	if isMetricExportV2Enabled() {
+		var ms []datadogV2.MetricSeries
+		ms, sl, sp = consumer.(*metrics.Consumer).All(exp.getPushTime(), exp.params.BuildInfo, tags)
+		ms = metrics.PrepareSystemMetrics(ms)
+
+		err = nil
+		if len(ms) > 0 {
+			exp.params.Logger.Debug("exporting native Datadog payload", zap.Any("metric", ms))
+			err = multierr.Append(
+				err,
+				exp.retrier.DoWithRetries(ctx, func(context.Context) error {
+					ctx = clientutil.GetRequestContext(ctx, exp.cfg.API.Key)
+					_, _, merr := exp.metricsAPI.SubmitMetrics(ctx, datadogV2.MetricPayload{Series: ms}, *datadogV2.NewSubmitMetricsOptionalParameters())
```

Comment:
I think we can just do
```suggestion
					_, _, merr := exp.metricsAPI.SubmitMetrics(ctx, datadogV2.MetricPayload{Series: ms})
```

if not using any optional parameters?